libass/ass_directwrite.c

```diff
@@ -314,11 +314,13 @@ static bool init_font_private_stream(FontPrivate *priv)
 
     hr = IDWriteFontFileLoader_CreateStreamFromKey(loader, refKey, keySize, &stream);
     if (FAILED(hr) || !stream) {
+        IDWriteFontFileLoader_Release(loader);
         IDWriteFontFile_Release(file);
         return false;
     }
 
     priv->stream = stream;
+    IDWriteFontFileLoader_Release(loader);
     IDWriteFontFile_Release(file);
```

Comment:
Ah this is an outdated commit (as GitHub says too). In final version there's a single cleanup section at the bottom, after outlining the common code to a separate function (and then they need to be if-checked, since the helper function may not initialize all of them successfully)